src/app/wallet/overview/overview.component.html

```diff
@@ -99,9 +99,10 @@
           <div class="item"><div class="value">2</div><div class="desc">Shipped</div></div>
         </div>
       </mat-card>
-    </div><-- .buy-orders -->
-    <app-coldstake></app-coldstake>
-    <app-stake></app-stake>
+    </div><!-- .buy-orders -->
+    <app-coldstake></app-coldstake> 
+    <app-stake *ngIf="!_coldstake.coldStakingEnabled"></app-stake>
```

Comment:
Set to only show if cold staking is not enabled or weight is > 0 on hot staking.